Tella/Data/Networking/Repositories/UwaziServerRepositories.swift

```diff
@@ -24,9 +24,9 @@ struct UwaziServerRepository: WebRepository {
     func twoFactorAuthentication(username: String,
                                   password: String,
                                   token: String,
-                                  serverURL: String) -> AnyPublisher<(UwaziLoginResult,HTTPURLResponse?), APIError> {
+                                  serverURL: String) -> AnyPublisher<(BoolResponse,HTTPURLResponse?), APIError> {
 
-        let call : AnyPublisher<(UwaziLoginResult,HTTPURLResponse?), APIError> = callReturnsHeaders(endpoint: API.twoFactorAuthentication((username: username, password: password,token: token, serverURL: serverURL)))
+        let call : AnyPublisher<(BoolResponse,HTTPURLResponse?), APIError> = callReturnsHeaders(endpoint: API.twoFactorAuthentication((username: username, password: password,token: token, serverURL: serverURL)))
```

Comment:
@rob-horizontal-iOS I think it's better to treat the header(`"Set-Cookie"`) here not in the view model

Reply:
@rob-horizontal-iOS Could you add the error messages in the `customErrorMessage` like "Invalid username or password" instead to pu it in the view model

Reply:
@dhekra-rouatbi  
- I moved the logic of token to [UwaziServerRepositories.swift](https://github.com/Horizontal-org/Tella-iOS/pull/80/files/2a0dc6816fd3b647a68f8d6b8cfa269077bb7e4b#diff-90473f13b84282abd89a4538f9cab91ac57caaabe6e65dbe872833d6a3adceef)
- CustomErrorMessage added for "Invalid username or password"